src/layouts/EditHomepage.jsx

```diff
@@ -1405,16 +1406,45 @@ const EditHomepage = ({ match }) => {
             {/* Isomer Template Pane */}
             {/* Notification */}
             {frontMatter.notification && (
-              <div className="bp-notification bg-secondary is-marginless">
-                <div className="bp-container">
-                  <div className="row">
-                    <div className="col">
-                      <div className="field has-addons bp-notification-flex">
-                        <div className="control has-text-centered has-text-white">
+              <div
+                id="notification-bar"
+                className={getClassNames(
+                  editorStyles,
+                  ["bp-notification", "is-marginless"],
+                  ["bg-secondary"]
+                )}
```

Comment:
could i clarify why we are getting the first two from `editorStyles` but not so for `bg-secondary`?

Reply:
`bg-secondary` only affects colours, and we're attempting to override the colours with site specific ones using our site colours hook - so I've opted to leave it out so that we don't run into any css specificity issues, especially since our imported styles make use of `!important` frequently